src/Common/WebTestCase.php

```diff
@@ -93,7 +93,7 @@ protected function loadFixtures(Client $client, array $fixtureClasses = [])
 
         // load all the fixtures
         foreach ($fixtureClasses as $class) {
-            $fixture = new $class();
+            $fixture = new $class($client);
             $fixture->load($database);
```

Comment:
since you are probably going to pass the container in the constructor I don't really see a reason why you would still pass the $database to the load method

Reply:
Won't do, as this is probably used in many test, can be done when everything uses entities, but maybe we should look into https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html at that point